pkg/env/readers.go

```diff
@@ -58,9 +96,9 @@ func devReader(fallback cfg.Reader) cfg.Reader { //nolint:deadcode,unused // Why
 	})
 }
 
-func testReader(fallback cfg.Reader, overrides map[string]interface{}) cfg.Reader {
+func testReader(fallback cfg.Reader, overrides *testOverrides) cfg.Reader {
```

Comment:
This function parameter name shadows the module-level variable, which is pretty bad for readability. Rename one of them?

Reply:
changing it to overrider